src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -719,60 +976,60 @@ CHIP_ERROR Instance::EnumerateAcceptedCommands(const ConcreteClusterPath & clust
 {
     using namespace Clusters::NetworkCommissioning::Commands;
 
-    constexpr CommandId acceptedCommandsListWiFi[] = {
-        ScanNetworks::Id, AddOrUpdateWiFiNetwork::Id, RemoveNetwork::Id, ConnectNetwork::Id, ReorderNetwork::Id,
-    };
-    constexpr CommandId acceptedCommandsListThread[] = {
-        ScanNetworks::Id, AddOrUpdateThreadNetwork::Id, RemoveNetwork::Id, ConnectNetwork::Id, ReorderNetwork::Id,
-    };
-
     if (mFeatureFlags.Has(Feature::kThreadNetworkInterface))
     {
-        for (const auto & cmd : acceptedCommandsListThread)
+        for (auto && cmd : {
+                 ScanNetworks::Id,
```

Comment:
Why is it better to have this list inline instead of in a constant?  I guess either way is OK, but just wondering whether there was a reason for the change.

Reply:
I'm emitting further allowed commands when the PDC feature is enabled, so having the array definitions outside of the `if (mFeatureFlags.Has(...))` makes the code harder to read. And once they're moved inside the if anyway, I didn't really see any benefit in assigning it into a named constant array first.